sdk/src/types/block/semantic.rs

```diff
@@ -147,11 +147,10 @@ impl TryFrom<u8> for TransactionFailureReason {
 }
 
 ///
-pub struct ValidationContext<'a> {
+pub struct SemanticValidationContext<'a> {
```

Comment:
maybe add a doc comment now?

Reply:
I think it's way too soon to add documentation on things that are 100% going to change, I can add a super generic one if you want but I don't really see the point

Reply:
no, leave it as it is then if you think that can't be properly described yet.